rules/zoom_operation_rules/zoom_sign_in_method_modified.py

```diff
@@ -0,0 +1,21 @@
+def rule(event):
+    operation_detail = event.get("operation_detail", "<NO_OPS_DETAIL>")
+    operation_flag = "Sign-in Methods  - Allow users to sign in with "
+    setting_flag = "from Off to On"
+    return (
+        event.get("action", "<NO_ACTION>") == "Update"
+        and event.get("category_type", "<NO_CATEGORY_TYPE>") == "Account"
+        and (operation_flag in operation_detail and setting_flag in operation_detail)
+    )
+
+
+def title(event):
+    # string manipulation to grab service that allows sign-in from the operation detail
+    # and clean it up a bit
+    service_detail = (
+        event.get("operation_detail", "<NO_OPS_DETAIL>").split("with")[1].split(":")[0].strip()
```

Comment:
This raised an exception locally if `operation_detail` is not found in event or if the operation_detail value is `Edit User user@email.com  - Change Type: from Basic to Licensed`

Reply:
oh hmmm
okay taking a look into this further

Reply:
update: couldn't possibly get the `edit user` thing because of lines 6-7.

Reply:
no worries! the suggestion was still helpful for me to put in the failsafes in case ops detail wasn't in the json!